src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -2049,10 +2100,19 @@ static int parse_args(vector<const char*>& args, std::ostream *err_msg,
       *err_msg << "rbd-nbd: --timeout is deprecated (use --io-timeout)";
     } else if (ceph_argparse_witharg(args, i, &cfg->format, err, "--format",
                                      (char *)NULL)) {
+    } else if (ceph_argparse_flag(args, i, "--all", (char *)NULL)) {
+      cfg->show_all = true;
     } else if (ceph_argparse_flag(args, i, "--pretty-format", (char *)NULL)) {
       cfg->pretty_format = true;
     } else if (ceph_argparse_flag(args, i, "--try-netlink", (char *)NULL)) {
       cfg->try_netlink = true;
+    } else if (ceph_argparse_flag(args, i, "--show-cookie", (char *)NULL)) {
+      cfg->show_cookie = true;
+    } else if (ceph_argparse_witharg(args, i, &cfg->cookie, "--cookie", (char *)NULL)) {
+      if (!is_valid_uuid(cfg->cookie)) {
```

Comment:
nit: Actually I don't see much value in this check. If it is not valid it will fail later anyway. But I don't mind much, you can leave it as it is now if other people think it is ok.

Reply:
Sure, will take care in next respin.

Reply:
Yeah, I'd drop this check and `is_valid_uuid()`.

Reply:
Sure